VTEX - Logistics API.json

```diff
@@ -4392,54 +4376,50 @@
                                     "unlimitedQuantity": {
                                         "type": "boolean",
-                                        "description": "When set as `true`, you make the SKU from the given warehouse permanently available for sales. No matter how many units are sold, the default quantity of `1000000` units does not decrease, and the store never runs out of stock. When set as `false`, every sold unit will decrease your inventory quantity. Note that:\r\n\r\n- Sending this field as `null` sets the value to `false`.\r\n\r\n- Not sending this field sets the value to `false`.\r\n\r\n- Sending this field as `true` overrules `quantity`.",
-                                        "nullable": true
-
+                                        "description": "When set as `true`, you make the SKU from the given warehouse permanently available for sales. No matter how many units are sold, the default quantity of `1000000` units does not decrease, and the store never runs out of stock. When set as `false`, every sold unit will decrease your inventory quantity. Note that:\r\n\r\n- Sending this field as `null` sets the value to `false`.\r\n\r\n- Not sending this field sets the value to `false`.\r\n\r\n- Sending this field as `true` overrules `quantity`."
                                     },
                                     "dateUtcOnBalanceSystem": {
                                         "type": "string",
-                                        "description": "Defines the corresponding moment to the given warehouse. It is useful due to the liberation of handling order reservations.\r\n\r\nWhen set as `null`, the value will be the date/time of the request. Its format is `DateTimeOffset`, as in `yyyy-mm-dd-Thh:mm:ss`. For example: `2023-03-15T00:52:16`.",
-                                        "example": "2023-03-15T00:52:16",
-                                        "nullable": true
+                                        "description": "Defines the date and time of the warehouse when the SKU was updated. This can be useful for liberating handling order [reservations](https://help.vtex.com/en/tutorial/how-the-reservation-works--tutorials_92), for example. The format is [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DDThh:mm:ss.ssZ. \r\nWhen the value is `null` or empty, the default will be the date and time of the when the request being made.",
```

Comment:
```suggestion
                                        "description": "Defines the date and time of the warehouse when the SKU was updated. This can be useful for liberating handling order [reservations](https://help.vtex.com/en/tutorial/how-the-reservation-works--tutorials_92), for example. The format is [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DDThh:mm:ss.ssZ. \r\nWhen the value is `null` or empty, the default will be the date and time of the request being made.",
```